composables/address-book.ts

```
@@ -0,0 +1,122 @@
+  ) => {
+    const entries = type ? getEntriesByType(type) : addressBook.value;
+
+    switch (sortBy) {
+      case 'name':
+        return entries.sort((a: AddressEntry, b: AddressEntry) => a.name.localeCompare(b.name, 'ja'));
+      case 'date':
+        return entries.sort((a: AddressEntry, b: AddressEntry) => new Date(b.updatedAt).getTime() - new Date(a.updatedAt).getTime());
+      case 'lastUsed':
+      default:
+        // 最終使用日でソート（使用されていないものは最後に）
+        return entries.sort((a: AddressEntry, b: AddressEntry) => new Date(b.lastUsedAt).getTime() - new Date(a.lastUsedAt).getTime());
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)